cnishina/protractor-lite
# WIP, not sure...

- How do I run a jasmine test with TypeScript?
- Should I have another ts_library for the spec file?
- How should I do the BUILD deps or tsconfig?

Running the current configuration fails because there are no specs
found.

```
>: bazel test lib/by:protractor_by_test

Starting local Bazel server and connecting to it...
INFO: Analysed target //lib/by:protractor_by_test (98 packages loaded).
INFO: Found 1 test target...
INFO: From Compiling TypeScript (devmode) //lib/by:protractor_by:

WARNING: your tsconfig.json file specifies options which are overridden by Bazel:
 - compilerOptions.target and compilerOptions.module are controlled by downstream dependencies, such as ts_devserver
 - compilerOptions.declaration is always true, as it's needed for dependent libraries to type-check
 - compilerOptions.noEmitOnError is always false under Bazel

FAIL: //lib/by:protractor_by_test (see /home/cnishina/.cache/bazel/_bazel_cnishina/c97a7eaf0d7d2d45f75abdaa2c711773/execroot/__main__/bazel-out/k8-fastbuild/testlogs/lib/by/protractor_by_test/test.log)
Target //lib/by:protractor_by_test up-to-date:
  bazel-bin/lib/by/protractor_by_test_loader.js
  bazel-bin/lib/by/protractor_by_test.sh
INFO: Elapsed time: 11.230s, Critical Path: 6.12s
INFO: 2 processes: 2 linux-sandbox.
INFO: Build completed, 1 test FAILED, 6 total actions
//lib/by:protractor_by_test                                              FAILED in 0.2s
  /home/cnishina/.cache/bazel/_bazel_cnishina/c97a7eaf0d7d2d45f75abdaa2c711773/execroot/__main__/bazel-out/k8-fastbuild/testlogs/lib/by/protractor_by_test/test.log

INFO: Build completed, 1 test FAILED, 6 total actions

>:  /home/cnishina/.cache/bazel/_bazel_cnishina/c97a7eaf0d7d2d45f75abdaa2c711773/execroot/__main__/bazel-out/k8-fastbuild/testlogs/lib/by/protractor_by_test/test.log

exec ${PAGER:-/usr/bin/less} "$0" || exit 1
Executing tests from //lib/by:protractor_by_test
-----------------------------------------------------------------------------
WARNING: source-map-support module not installed.
    Stack traces from languages like TypeScript will point to generated .js files.
    Set install_source_map_support = False in //lib/by:protractor_by_test to turn off this warning.

Randomized with seed 18155
Started

No specs found
Finished in 0.001 seconds
Incomplete: No specs found
